### PR TITLE
disable codeql on terraform-only changes

### PR DESCRIPTION
**Pull request by bobcallaway**

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

**Review by k4leung4 — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ on:
   pull_request:
     branches:
       - main
+    paths-ignore:
+      - 'terraform/**'
```
Comment (cpanato) [inline .github/workflows/codeql-analysis.yml:11]: humm that will not run and the check that is required will not be valid and will block

we should use the `dorny/paths-filter@v2` to filter then that will properly skip the job

**Review by cpanato — CHANGES_REQUESTED** [review]
sorry double check that

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (bobcallaway) [reply .github/workflows/codeql-analysis.yml:11]: hmm, its worked across other workflows in this repo....... I guess I don't understand the difference?

Comment (cpanato) [reply .github/workflows/codeql-analysis.yml:11]: Oh really? That is good news then GitHub changed that behavior nice

**Review by cpanato — APPROVED** [review]

**Pull request merged**